cupy/core/_kernel.pyx

```diff
@@ -507,6 +500,9 @@ cdef class ElementwiseKernel:
         """
         cdef function.Function kern
         cdef Py_ssize_t size
+        cdef int i
```

Comment:
`i` should be `Py_ssize_t` becaues it comes from `self.nin`.
(Btw I don't think the explicit declaration is needed.)